crates/sui-graphql-e2e-tests/tests/call/owned_objects.exp

```diff
@@ -30,7 +30,7 @@ task 5 'create-checkpoint'. lines 58-58:
 Checkpoint created: 1
 
 task 6 'view-checkpoint'. lines 60-60:
-CheckpointSummary { epoch: 0, seq: 1, content_digest: 3roqGGtXUFwWgryXd47b8excsteDJyFXKoKUmFEcUbpD,
+CheckpointSummary { epoch: 0, seq: 1, content_digest: 94FSxnwDKDpJ8txUBNcnxU75aathRbysGuihNUchZokV,
```

Comment:
I'm surprised that this test needed updating given that the test logic didn't seem to change -- why is that?